packages/blade/src/components/List/List.tsx

```diff
@@ -129,7 +128,7 @@ const _List = ({
   );
 };
 
-const List = assignWithoutSideEffects(_List, { componentId: ComponentIds.List });
+const List = assignWithoutSideEffects(_List, { componentId: MetaConstants.List });
 
```

Comment:
Why this change 🤔? we usually use ComponentIds here no?

Reply:
This was becoming too confusing, using 2 constants for the same thing. We should use the same naming convention for component name on meta data as well as id. Otherwise we'll maintain 2 constants for every component.